lib/correios/cep/web_service.rb

```diff
@@ -23,43 +24,12 @@ def initialize
       end
 
       def request(zipcode)
-        http = build_http
-
-        request = build_request(zipcode)
-        Correios::CEP.log_request(request, uri.to_s)
-
-        response = http.request(request)
-        Correios::CEP.log_response(response)
-
-        http.finish if http.started?
-
-        response.body
-      end
-
-      private
-
-      attr_reader :uri, :proxy_uri
-
-      def build_http
-        Net::HTTP.start(
-          uri.host,
-          uri.port,
-          proxy_uri.host,
-          proxy_uri.port,
-          nil,
-          nil,
-          use_ssl: true,
-          verify_mode: OpenSSL::SSL::VERIFY_NONE,
-          open_timeout: Correios::CEP.request_timeout,
-          read_timeout: Correios::CEP.request_timeout
-        )
-      end
-
-      def build_request(zipcode)
-        request = Net::HTTP::Post.new(uri.path)
-        request['Content-Type'] = CONTENT_TYPE_HEADER
-        request.body = BODY_TEMPLATE % { zipcode: zipcode }
-        request
+        HTTP
+          .use(logging: {logger: Logger.new(STDOUT)})
```

Comment:
Logger configuration is not being used.

Reply:
What do you mean? I tested here and when the request is made and received the log prints

Reply:
You should use the logger is set on configuration. For example: https://github.com/prodis/correios-cep#configuration-example

In your implementation you fixed the logger to STDOUT.